pkg-new/hostutils/initialize.go

```diff
@@ -39,6 +40,31 @@ func (h *HostUtils) ConfigureHost(ctx context.Context, rc runtimeconfig.RuntimeC
 		}
 	}
 
+	h.logger.Debugln("checking for semanage binary in $PATH")
```

Comment:
nit: how about moving these two selinux related changes to an selinux.go and add them as methods on the host utils interface so they can be mocked?